DOCS-5291 React Native docs improvements (#17815)
DataDog/documentation: content/en/real_user_monitoring/reactnative/codepush.md

Datadog recommends using `@datadog/mobile-react-native-code-push` in your app and the [datadog-ci][3] `react-native codepush` command to upload your source maps. This ensures that the `version` is consistent in both reported crashes and uploaded source maps.

> If you experience any issue setting up the Datadog SDK with codepush you can check out our [example application][6].

Reviewer: ```suggestion
If you experience any issues setting up the Datadog SDK with codepush, you can see our [example application][6] as a reference.
```
